test/unit/StakingPool/setProducts.js

```diff
@@ -8,6 +8,7 @@ const ProductTypeFixture = {
   claimMethod: 1,
   gracePeriodInDays: 7,
 };
+const GLOBAL_MIN_PRICE_RATIO = 100;
```

Comment:
[minor] Do you think it would make sense to fetch it from the contract instead? I'm thinking that if we change it at any point in the future the tests may start failing.